how-to/how-to-use-virtio-fs-with-kata.md

```diff
@@ -21,7 +21,7 @@ This document describes how to get Kata Containers to work with virtio-fs.
 
 ## Pre-requisites
 
-* This feature currently requires the host to have hugepages support enabled. Enable this with the `sysctl vm.nr_hugepages=1024` command on the host.
+* This feature currently requires the host to have huge pages support enabled. Enable this with the `sysctl vm.nr_hugepages=1024` command on the host.
```

Comment:
hmm, OK, I think I can live with this, but it is `hugepages` I think in every config file and command line etc....

Reply:
NACK.  Lets add hugepages to the dictionary.

Reply:
You can't see it yet, but "done" ;)

Reply:
And branch updated ;)